DeploymentCloud/Deployment.DataX/Samples/flows/iotsample-product.json

```diff
@@ -412,7 +417,10 @@
 			],
 			"jobconfig" : {
 				"jobNumExecutors" : "4",
-				"jobExecutorMemory" : "1024"
+				"jobExecutorMemory" : "1024",
+				"jobDatabricksAutoScale" : true,
```

Comment:
Do these new params work for HDinsight?

Reply:
yes.. these params work for hdinsight too